frontend/public/components/app-contents.tsx

```diff
@@ -438,7 +438,7 @@ const AppContents_: React.FC<AppContentsProps> = ({ activePerspective }) => {
               path="/monitoring/alerts"
               exact
               loader={() =>
-                import('./monitoring' /* webpackChunkName: "monitoring" */).then(
+                import('./monitoring/alerting' /* webpackChunkName: "monitoring" */).then(
```

Comment:
Should we change the webpack chunk name too?

Reply:
👍 Changed to `alerting`